src/app/docs/tabs/tabs-docs.component.ts

```diff
@@ -1,9 +1,77 @@
+// tslint:disable:member-ordering
 import {
+  ChangeDetectionStrategy,
   Component
 } from '@angular/core';
 
+import {
+  SkyDocsDemoControlPanelChange
+} from '@skyux/docs-tools';
+
 @Component({
   selector: 'app-tabs-docs',
-  templateUrl: './tabs-docs.component.html'
+  templateUrl: './tabs-docs.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class TabsDocsComponent { }
+export class TabsDocsComponent {
+
+  public demoSettings: any = {};
+
+  public set includeCounts(value: boolean) {
+    this._includeCounts = value;
+  }
+
+  public get includeCounts(): boolean {
+    return this._includeCounts;
```

Comment:
Do you need to return a default `boolean` value in the getter? Otherwise, I don't think we need the get/set.